tests/unit/amo/components/TestGetFirefoxButton.js

```diff
@@ -136,57 +136,55 @@ describe(__filename, () => {
       });
 
       // See: https://docs.google.com/document/d/1vXpEg_ypqr-eiXu6pWBDiyQWwf_rxYhHAGnsp3qwpCo/edit?usp=sharing
-      it('sets utm_campaign to non-fx-button and utm_term to the expected value for RTAMO (useNewVersion = false)', () => {
+      it('passes the expected URL params on the download link (useNewVersion = false)', () => {
         const guid = 'some-guid';
         const addon = createInternalAddonWithLang({ ...fakeAddon, guid });
         const root = render({
           addon,
           buttonType,
           store,
+          overrideQueryParams: {
+            experiment: EXPERIMENT_CONFIG.id,
+            variation: VARIANT_CURRENT,
+          },
           useNewVersion: false,
         });
 
-        const utmContent = `rta:${encode(addon.guid)}`;
-        const utmTerm = getDownloadTerm({
-          addonId: addon.id,
-          variant: VARIANT_CURRENT,
-        });
-        const expectedHref = `${DOWNLOAD_FIREFOX_BASE_URL}${makeQueryStringWithUTM(
-          {
-            utm_campaign: 'non-fx-button',
-            utm_content: utmContent,
-            utm_term: utmTerm,
-          },
-        )}`;
+        const expectedHref = `${DOWNLOAD_FIREFOX_BASE_URL}?experiment=${
+          EXPERIMENT_CONFIG.id
+        }&variation=${VARIANT_CURRENT}&utm_campaign=non-fx-button&utm_content=rta%3A${encode(
+          addon.guid,
+        )}&utm_medium=referral&utm_source=addons.mozilla.org&utm_term=amo-fx-cta-${
+          addon.id
+        }-${VARIANT_CURRENT}`;
 
         expect(root.find('.GetFirefoxButton-button')).toHaveProp(
           'href',
           expectedHref,
         );
       });
 
-      it('sets utm_campaign to non-fx-button and utm_term to the expected value for RTAMO (useNewVersion = true)', () => {
+      it('passes the expected URL params on the download link (useNewVersion = true)', () => {
         const guid = 'some-guid';
         const addon = createInternalAddonWithLang({ ...fakeAddon, guid });
         const root = render({
           addon,
           buttonType,
           store,
+          overrideQueryParams: {
+            experiment: EXPERIMENT_CONFIG.id,
+            variation: VARIANT_NEW,
+          },
           useNewVersion: true,
         });
 
-        const utmContent = `rta:${encode(addon.guid)}`;
-        const utmTerm = getDownloadTerm({
-          addonId: addon.id,
-          variant: VARIANT_NEW,
-        });
-        const expectedHref = `${DOWNLOAD_FIREFOX_BASE_URL}${makeQueryStringWithUTM(
-          {
-            utm_campaign: 'non-fx-button',
-            utm_content: utmContent,
-            utm_term: utmTerm,
-          },
-        )}`;
+        const expectedHref = `${DOWNLOAD_FIREFOX_BASE_URL}?experiment=${
+          EXPERIMENT_CONFIG.id
+        }&variation=${VARIANT_NEW}&utm_campaign=non-fx-button&utm_content=rta%3A${encode(
+          addon.guid,
+        )}&utm_medium=referral&utm_source=addons.mozilla.org&utm_term=amo-fx-cta-${
```

Comment:
To avoid this I usually use an array like:

```js
const queryString = [
  `variation=${VARIANT_NEW}`,
  'utm_medium=referral',
  // ...
].join('&');

const expectedHref = `${DOWNLOAD_FIREFOX_BASE_URL}?${queryString}`;
```